preprocess/raven_preprocess/type_guess_util.py

```diff
@@ -9,6 +10,22 @@
 from raven_preprocess.column_info import ColumnInfo
 from raven_preprocess.basic_utils.basic_err_check import BasicErrCheck
 
+date_re = re.compile(r'[\d]+\.[\d]+\.[\d]+')
+not_date_re = re.compile(r'-?[.\d]+')
+
+def parse_date(val, year):
+    if not isinstance(val, str):
+        if not year and val < 1600 or val > 2100:
```

Comment:
- Please move min/max date restrictions 1600 and 2100 to some type of settings/configuration
    - Can slack or chat to what may be appropriate for these settings